app/components/Albumsssv2/Songs/page.tsx

```diff
@@ -0,0 +1,29 @@
+
+import { albumsState } from "@/app/states";
+import { useState, useEffect } from "react";
+import { useRecoilState } from "recoil";
+import Albumsartistsv2 from "../Albumsartists/page";
+import { useSearchParams } from "next/navigation";
+
+
+const AlbumSongs = () => {
+    let params = useSearchParams();
+    let albumParam = params.get('album');
+
+    const [albumName, setAlbumName] = useState('')
+    const [myAlbum, setMyAlbum] = useRecoilState(albumsState)
```

Comment:
is recoil surely needed here , i dont think soo

Reply:
yes, we need this